source/_templates/base.html

```diff
@@ -75,6 +75,18 @@
 
     {#- Custom front matter #}
     {%- block extrahead -%}{%- endblock -%}
+
+        
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
No need for this comment to end up in the rendered markup.
```suggestion
    {#- Global site tag (gtag.js) - Google Analytics -#}
```